conu/docker/core.py

```diff
@@ -147,3 +148,22 @@ def copy_to(self, src, dest):
     def copy_from(self, src, dest):
         self.start()
         run_cmd("docker cp %s:%s %s" % (self.tag, src, dest))
+
+    def check_files_exist(self, files):
+        """
+        Function returns
+        :param files:
+        :return: True if all files exists
+                 False if at least one does not exist
+        """
+        found = True
+        if isinstance(files, str):
+            files = [files]
+        if files:
+            for f in files:
+                logger.debug("Check if %s file is present in container." % f)
+                ret_val = self.execute('ls %s' % f)
```

Comment:
I'll be honest that I'm not sure about the implementation: I would prefer mounting the whole container FS and then just perform these kind of operations on local FS rather then go wild in a container -- what if there is no `ls` in the container? Let's talk about this on the hackday.